lib/rag-engines/index.ts

```diff
@@ -42,10 +42,7 @@ export class RagEngines extends Construct {
     const tables = new RagDynamoDBTables(this, "RagDynamoDBTables");
 
     let sageMakerRagModels: SageMakerRagModels | null = null;
-    if (
-      props.config.rag.engines.aurora.enabled ||
-      props.config.rag.engines.opensearch.enabled
-    ) {
+    if (props.config.rag.crossEncodingEnabled) {
```

Comment:
You were right, `props.config.llms.enableSagemakerModels` is better.

Reply:
Updated the condition.